test/e2e/grid.test.js

```diff
@@ -60,6 +60,16 @@ describe('vl-grid', async () => {
     await assert.eventually.isTrue(stackedGrid.isStacked());
   });
 
+  it('men kan een grid stacked large maken', async () => {
+    const stackedGrid = await vlGridPage.getStackedLargeGrid();
+    await assert.eventually.isTrue(stackedGrid.isStackedLarge());
+  });
+
+  it('men kan een grid stacked small maken', async () => {
+    const stackedGrid = await vlGridPage.getStackedSmallGrid();
+    await assert.eventually.isTrue(stackedGrid.isStackedSmall());
+  });
```

Comment:
Ik test vaak voor de zekerheid ook nog dat een niet stacked large grid false geeft bij de assert, maar dat leek in dit geval nergens uitgevoerd. Voorlopig prima voor mij.